test/OverlayTriggerSpec.js

```diff
@@ -111,6 +111,33 @@ describe('<OverlayTrigger>', () => {
     wrapper.assertSingle('div.test');
   });
 
+  it('Should show after mouseover trigger', (done) => {
+    const clock = sinon.useFakeTimers();
+
```

Comment:
followed this old closed PR example for using fake timers + clock via Sinon react-bootstrap#4573